hack/lib/catalogsource.bash

```diff
@@ -129,7 +129,7 @@ function add_user {
   kubectl create secret generic htpass-secret \
     --from-file=htpasswd="$(pwd)/users.htpasswd" \
     -n openshift-config \
-    --dry-run -o yaml | kubectl apply -f -
+    --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
Can we back out those changes into their own PR please? I've done them once but we had issues with different `oc` versions for peeps.

Putting them in their own PR at least makes them easily revertable.

Reply:
thanks for catching that -- backed out this change in 23233e0 and will create a separate PR